graderutils/graderunittest.py

```diff
@@ -168,11 +174,13 @@ def run_test_suite_in_named_module(module_name):
     Return a PointsTestResult containing the results.
     """
     loader = unittest.defaultTestLoader
-    test_module = importlib.import_module(module_name)
-    test_suite = loader.loadTestsFromModule(test_module)
-    # Redirect output to string stream and increase verbosity
-    runner = PointsTestRunner(stream=io.StringIO(), verbosity=2)
-    result = runner.run(test_suite)
+    # Module output must be suppressed during import and run, since grading json is printed to stdout as well
+    with contextlib.redirect_stdout(None):
```

Comment:
In my testing, the grading did not get disrupted if the student prints to stderr. Though, if they print a lot, it will increase the amount of data sent back to the mooc-grader and a-plus and a-plus may fail to save it to the database.

Reply:
#22